archtec/scripts/common.lua

```diff
@@ -41,6 +42,15 @@ function core.kick_player(player_name, reason)
 	return core.disconnect_player(player_name, reason)
 end
 
+function core.kick_inactive_player(player_name)
```

Comment:
I could just make everything inside one function that already existed, but I was not sure about code conventions that would be preferable for this repo, so special method it is!